src/backend/distributed/transaction/worker_transaction.c

```diff
@@ -69,6 +69,58 @@ SendCommandToWorkers(TargetWorkerSet targetWorkerSet, char *command)
 
 
 /*
+ * SendBareCommandListToWorkers sends a list of commands to a set of target
+ * workers in serial. Commands are committed immediately: new connections are
+ * always used and no transaction block is used (hence "bare"). The connections
+ * are made as the extension owner to ensure write access to the Citus metadata
+ * tables. Primarly useful for INDEX commands using CONCURRENTLY.
+ */
+void
+SendBareCommandListToWorkers(TargetWorkerSet targetWorkerSet, List *commandList)
```

Comment:
Elsewhere in the code we already use `COMMIT_PROTOCOL_BARE` to describe commands sent without any sort of transaction wrapping them, so it's out of analogy to that. I might leave this the way it is.